src/services/tourettes.js

```diff
@@ -62,12 +62,22 @@ function getProfanities(amount) {
   return profanityReply.join(' ');
 }
 
+function isInt(data) {
```

Comment:
No you're not missing anything, I wasn't sure what version of JS node supported and I found that on stack overflow complete with a jsperf test where it was at least a hundred times faster than comparable tests and I sort of stuck with it.

I'd say let's use internal methods if possible, I just forgot about it until we started to discuss it! :)